tests/api/garage/carId.spec.js

```diff
@@ -0,0 +1,32 @@
+
+import {expect, loginApi as test} from "../../../src/fixtures/loginApi.js";
+
+test.describe("Get car current User", ()=> {
+
+    test(`Get car current User by id`, async({carsController, createCar })=>{
+            
+        const requestCarId = await carsController.getCarCurrentUser(createCar.data.id)
+        const responseCarIdBody = await requestCarId.json()
+
+        console.log('currentUserCars:', createCar);      
+
+        expect(responseCarIdBody).toEqual({
+            "status": "ok",
+            "data": {
+            "id": createCar.data.id,
+            "carBrandId": createCar.data.carBrandId,
+            "carModelId": createCar.data.carModelId,
+            "initialMileage": createCar.data.initialMileage,
+            "updatedMileageAt": expect.any(String),
+            "carCreatedAt": expect.any(String),
```

Comment:
you can check if carCreatedAt has not changed

and updatedMileageAt should be also updated 

use some date library for that